src/nodes.coffee

```diff
@@ -4848,25 +4895,46 @@ mergeLocationData = (locationDataA, locationDataB) ->
 # location data object that encompasses the location data of both nodes. So the
 # new `start` value will be the earlier of the two nodes’ `start` values, the
 # new `end` value will be the later of the two nodes’ `end` values, etc.
-mergeAstLocationData = (nodeA, nodeB) ->
+mergeAstLocationData = (nodeA, nodeB, {justLeading, justEnding} = {}) ->
```

Comment:
Do you mind adding a comment explaining `justLeading` and `justEnding` and when they should be used?

Reply:
Sure, added to the existing function comment and added tests for `mergeAstLocationData()` to `test/abstract_syntax_tree_location_data.coffee`